contracts/base/Dispatcher.sol

```diff
@@ -281,23 +281,18 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, V4SwapRout
     /// @return output The resultant recipient for the command
     function map(address recipient) internal view returns (address) {
         if (recipient == Constants.MSG_SENDER) {
```

Comment:
Once we have `main` in can you delete `MSG_SENDER` `ADDRESS_THIS` and `CONTRACT_BALANCE` from `Constants.sol` and just use the ones that are in `ActionsConstants.sol` in v4?